### PR TITLE
Live merge. Enterprise to master.

### PR DESCRIPTION
**Pull request by AdamZielinski-Devtactics**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -15,6 +16,13 @@ gem 'travis-support',   git: 'https://github.com/travis-ci/travis-support'
 gem 'metriks',                 git: 'https://github.com/travis-ci/metriks'
 gem 'metriks-librato_metrics', git: 'https://github.com/travis-ci/metriks-librato_metrics'
 
+gem 'redis'
+gem 'redis-namespace'
+gem 'rake', '~> 13.0.6'
+
+gem 'jemalloc', git: 'https://github.com/travis-ci/jemalloc-rb', branch: 'jemalloc-v-3-new-rake'
+
+gem 'sentry-raven'
```
Comment (gbarc-dt) [inline Gemfile:25]: sentry-raven is now sentry-ruby

File: Makefile2
Comment (gbarc-dt) [inline Makefile2:1]: this one we don't need :P

Comment (AdamZielinski-Devtactics) [reply Makefile2:1]: Fixed.

File: .travis.yml [hunk 2]
```diff
@@ -31,3 +27,4 @@ jobs:
       install: skip
       script: make ship
```
Comment (GbArc) [inline .travis.yml:28]: add dist: focal

File: Makefile2
Comment (GbArc) [inline Makefile2:1]: this file isn't needed

**Review by GbArc — CHANGES_REQUESTED** [review]

**Comment by DominikAlberski** [comment]
Same as #56 merged in #57

**Pull request closed without merging**